lib/smart_proxy_dns_infoblox/dns_infoblox_main.rb

```diff
@@ -5,7 +5,7 @@ class Record < ::Proxy::Dns::Record
     def initialize(host, connection, ttl, dns_view)
       @connection = connection
       @dns_view = dns_view
-      super(ENV['INFOBLOX_DNS_RESOLVER'] || host, ttl)
```

Comment:
Can you also remove the mention of this var from the example config?

Reply:
Right, good idea.